.github/workflows/deploy-orch-build-and-promote.yml

```diff
@@ -0,0 +1,19 @@
+name: Deploy Orchestration build and promote
+
+permissions:
+  id-token: write
+  contents: read
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
```

Comment:
We need to remove this. This would enable someone to deploy a branch to production, bypassing the review process